README.md

```diff
@@ -122,16 +124,16 @@ export class AppComponent {
 
 ## Usage & Demo
 
-Main source of API documentation and usage scenarios available here:
+Main source of API documentation and usage scenarios are available here:
 [https://valor-software.com/ngx-bootstrap/](https://valor-software.com/ngx-bootstrap/)
 
 Additionally you can find demos and docs deployed from latest code with <a href="https://ngx-bootstrap.surge.sh/">angular@latest</a> and <a
   href="https://ngx-bootstrap-latest.surge.sh/">angular@next</a>
 
-Server side rendered version of this documentation available <a href="https://ngx-universal.herokuapp.com/">here</a> served with Angular universal and [nest.js](https://nestjs.com/)
+Server side rendered version of this documentation is available <a href="https://ngx-universal.herokuapp.com/">here</a> served with Anguar universal and [nest.js](https://nestjs.com/)
```

Comment:
Anguar? Oh...

Reply:
That is a legit error, I could have fixed that. Its a typo that occurred randomly. Without me having an idea about it. Just because you are a reviewer, you can't taunt someone like that